src/Drupal/MediaTrait.php

```
@@ -84,6 +87,38 @@ public function mediaCreate(string $media_type, TableNode $table): void {
+   *
+   * @code
+   * Given the following image media with fields:
+   *   | field | media_1              | media_2              |
+   *   | name  | [TEST] Image 1       | [TEST] Image 2       |
+   *   | field_media_image | image1.jpg | image2.jpg         |
+   * @endcode
+   */
+  public function mediaCreateWithFields(string $bundle, TableNode $table): void {
+    $entities = $this->helperTransposeVerticalTable($table);
+
+    foreach ($entities as $entity_data) {
```

PHPMD (2.15.0)

The variable $entity_data is not named in camelCase. (undefined)

(CamelCaseVariableName)